model/Core/Vocabularies/RelationshipType.md

```diff
@@ -24,7 +24,7 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - availableFrom:  This relationship is used to identify additional suppliers where an artifact is available from.
 - buildDependency: Every `to` Element is a build dependency of the `from` Element
 - buildTool: Build tool used to build an Element. This may be used to describe the build tool of a Build instance
-- coordinatedBy: (Security) Used to identify the vendor, researcher, or consumer agent performing coordination for a vulnerability
+- coordinatedBy: (Security) The `to` Agent(s) (vendor, researcher, or consumer agent) has performed coordination for a `from` Vulnerability element
```

Comment:
When it's not possible to make the documentation match the `from` .... `to` ordering, I added an example at the end just to make it extra clear, so I would append "(`from` coordinatedBy `to`)" to the end of this line

Reply:
The "from" Vulnerability is coordinatedBy the "to" Agent(s) (vendor, researcher, or consumer agent)